modules/ensemble/lib/widget/loading_container.dart

```diff
@@ -40,16 +42,46 @@ class LoadingContainer extends StatefulWidget
   Map<String, Function> setters() {
     return {
       'isLoading': (value) => _controller.isLoading = Utils.optionalBool(value),
-      'useShimmer': (value) =>
-          _controller.useShimmer = Utils.optionalBool(value),
-      'defaultShimmerPadding': (value) =>
-          _controller.defaultShimmerPadding = Utils.getInsets(value),
-      'baseColor': (color) => _controller.baseColor = Utils.getColor(color),
-      'highlightColor': (color) =>
-          _controller.highlightColor = Utils.getColor(color),
       'widget': (widget) => _controller.widget = widget,
       'loadingWidget': (loadingWidget) =>
           _controller.loadingWidget = loadingWidget,
+      'baseColor': (color) => _controller.baseColor = Utils.getColor(color),
+      'highlightColor': (color) =>
+          _controller.highlightColor = Utils.getColor(color),
+      'defaultShimmerPadding': (value) => _controller.padding =
+          Utils.getInsets(value), // Backward compatibility
+      'shimmerOptions': (options) {
+        if (options is Map<String, dynamic>) {
```

Comment:
useShimmer should be outside shimmerOptions. Keep it where it was please. User first sets the useShimmer to true and then specifies the shimmerOptions for it. It was fine where it was, fix it please.